org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/managers/ProjectsManagerTest.java

```diff
@@ -172,4 +181,101 @@ public void testResourceFilters() throws Exception {
 		}
 	}
 
+	@Test
+	public void testImportMavenSubModule() throws IOException, OperationCanceledException, CoreException {
+		File from = new File(getSourceProjectDirectory(), "maven/multimodule");
+		Path workspaceDir = Files.createTempDirectory("mavenMultiModule");
+
+		try {
+			Path projectDir = Files.createDirectory(workspaceDir.resolve("TheMavemMultiModuleProject"));
+			FileUtils.copyDirectory(from, projectDir.toFile());
```

Comment:
In test class, we already have a utility method `copyFiles("maven/multimodule", true)` to copy sample project to a temp working directory, is there any reason to not use it in your test case?